src/actions/vm.js

```diff
@@ -307,6 +306,17 @@ export function updateIcons ({ icons }) {
   }
 }
 
+export function setVmConsoles ({ vmId, consolesList, selectedProtocol }) {
+  return {
+    type: SET_VM_CONSOLES,
+    payload: {
+      vmId,
+      consoles: consolesList,
+      selectedProtocol,
```

Comment:
Again regarding naming:
Sometimes we use protocol and sometimes console which actually it's the same meaning in our code. So we need to be more persistent.
For example, instead of selectedProtocol here, we can have selectedConsole (on all places in code).